core/src/main/java/bisq/core/payment/payload/RevolutAccountPayload.java

```diff
@@ -105,7 +110,34 @@ public static RevolutAccountPayload fromProto(protobuf.PaymentAccountPayload pro
 
     @Override
     public String getPaymentDetails() {
-        return Res.get(paymentMethodId) + " - " + Res.getWithCol("payment.account.userName") + " " + getUserName();
+        Tuple2<String, String> tuple = getLabelValueTuple();
+        return Res.get(paymentMethodId) + " - " + tuple.first + ": " + tuple.second;
+    }
+
+    private Tuple2<String, String> getLabelValueTuple() {
```

Comment:
This is kind of a mess, but seems correct.

Reply:
Yes, not great all those special cases. Could be done mor elegantly for sure but this feature ate already much more time as expected, so tried to get it fixes as quickly I could ;-)